higlass/client.py

```diff
@@ -1,3 +1,4 @@
+import collections
```

Comment:
Is this used anywhere? I don't see it in this changeset?

Reply:
Oh, good catch. I used to use it to detect `dict` but I removed support for it.